src/components/CodeExample.md

```diff
@@ -0,0 +1,35 @@
+CodeExample - Endpoints
+
+```jsx
+import Table from "./Table";
+import { PALETTE } from "../shared";
+
+<CodeExample title="Endpoints">
```

Comment:
Is this component for something like this?

![image](https://user-images.githubusercontent.com/1668327/67042110-2df20400-f0f5-11e9-97f1-e31a1e200ee1.png)

If I saw a component called "CodeExample", I'd expect it to show a code snippet, like:

```js
for (let i = 0; i < 10; i++) {
  console.log(i);
}
```

If this example is just showing a title, then I'd name it something else, like "TableContainer" or "TableHeader" or something.

Reply:
This one is for the ones that appear on the right column in API Ref. _However_, closing this PR because we want to keep all the DOCs related component in stellar.org for now until the new design is done for stellar.org.